src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,12 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const category = createSlice({
+  name: '',
+  initialState: [],
+  reducers: {
+    checkStatus: () => 'under construction',
+  },
+});
```

Comment:
- [ ] Kindly take not that for proper identification, the name is not supposed to be empty. ℹ️ Kindly give it a descriptive name, maybe `categories`. 

- [ ] The logic for the category state is missing as well, kindy implement it. 👍 

```example
reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
```
```

![Uploading Untitled.png…]()